ui/src/app/app-routing.module.ts

```diff
@@ -150,7 +153,7 @@ const routes: Routes = [
                     data: {
                       pathElementForTitle: 'nbName',
                       breadcrumb: BreadcrumbType.Notebook,
-                      helpContent: 'preview',
+                      helpContent: 'notebookStorage',
```

Comment:
Perhaps this would be explained by the other comments, but why is the new property not set here?

Reply:
<img width="1680" alt="Screen Shot 2020-05-04 at 3 02 55 PM" src="https://user-images.githubusercontent.com/1847690/81013589-62c36f80-8e29-11ea-9824-3d5a619bd736.png">
`height: auto` behaves strangely when we're actually displaying the notebook. The content of the div is large enough that it ought to make the container div grow, but it's not doing that.